bokeh/models/widgets/buttons.py

```diff
@@ -121,24 +124,22 @@ class Toggle(AbstractButton):
     label = Override(default="Toggle")
 
     active = Bool(False, help="""
-    The initial state of a button. Also used to trigger ``on_click`` event
-    handler.
+    The state of the toggle button.
     """)
 
     def on_click(self, handler: Callable[[bool], None]) -> None:
         """ Set up a handler for button state changes (clicks).
-
```

Comment:
We use blank lines between summary/args/returns all throughout the codebase, so these should be restored just for consistency